### PR TITLE
Disable 'open terminal here' command for unsaved documents

### LINKED ISSUE #119: Disable 'open terminal here' menu item for unsaved documents
1. Open Sublime Text
2. File / New File
3. Right click in text editor, 'Open terminal here...'

Expected: Menu item 'Open terminal here' is greyed out (disabled)

What actually happens: You get a popup error "Terminal: No place to open terminal to"

### PR DESCRIPTION
**Pull request by hickford**
Fixes https://github.com/wbond/sublime_terminal/issues/119

### CODE REVIEW

**Comment by hickford** [comment]
Works for me in Sublime Text 3

File: Terminal.py [hunk 1]
```diff
@@ -165,6 +165,9 @@ def run(self, paths=[], parameters=None):
 
         self.run_terminal(path, parameters)
 
+    def is_enabled(self):
+        return bool(self.window.active_view() and self.window.active_view().file_name())
```
Comment (twolfson) [inline Terminal.py]: Should we add this to other commands (e.g. `OpenTerminalProjectFolderCommand`) and thus probably to `TerminalCommand`?

Comment (twolfson) [inline Terminal.py]: It should probably be calling `self.get_path(paths) and self.window.folders()` (verify file has a path and there is at least 1 open folder in the sidebar)

Comment (twolfson) [inline Terminal.py]: Also, can we update this `is_enabled` to use `get_path()` instead? It seems like they share the same logic

**Comment by braver** [comment]
@hickford the Terminal package is now managed at https://github.com/SublimeText/Terminal, and the latest release incorporates your suggestion via https://github.com/SublimeText/Terminal/commit/b94e8e8456ced26e37846f26bb9d157a867a3d34

**Pull request closed without merging**